diff --git a/browser/components/preferences/preferences.js b/browser/components/preferences/preferences.js
index e95831780d00111f27bb4ad78a6ed8ec39cf886e..676a8154ed38945ba7c01a4638c95e3742b0e6b7 100644
--- a/browser/components/preferences/preferences.js
+++ b/browser/components/preferences/preferences.js
@@ -132,6 +132,7 @@ ChromeUtils.defineLazyGetter(this, "gSubDialog", function () {
       styleSheets: [
         "chrome://browser/skin/preferences/dialog.css",
         "chrome://browser/skin/preferences/preferences.css",
+        "chrome://browser/skin/preferences/zen-preferences.css",
       ],
       resizeCallback: async ({ title, frame }) => {
         // Search within main document and highlight matched keyword.
@@ -437,6 +438,8 @@ const CONFIG_PANES = Object.freeze({
   tabsBrowsing: {
     l10nId: "tabs-browsing-section",
     groupIds: [
+      "defaultBrowserHome",
+      "startupHome",
       "browserLayout",
       "tabs",
       "pageNavigation",
@@ -477,7 +480,12 @@ function register_module(categoryName, categoryObject) {
       }
       this._initted = true;
       let template = document.getElementById("template-" + categoryName);
-      if (template && !srdSectionPrefs.all) {
+      if (
+        template &&
+        (!srdSectionPrefs.all ||
+          categoryName.startsWith("paneZen") ||
+          categoryName.startsWith("paneEpoca"))
+      ) {
         // Replace the template element with the nodes inside of it.
         template.replaceWith(template.content);
 
@@ -522,6 +530,11 @@ function init_all() {
   register_module("paneSearch", gSearchPane);
   register_module("panePrivacy", gPrivacyPane);
   register_module("paneContainers", gContainersPane);
+  register_module("paneZenLooks", gZenLooksAndFeel);
+  register_module("paneZenTabManagement", gZenWorkspacesSettings);
+  register_module("paneZenCKS", gZenCKSSettings);
+  register_module("paneZenMarketplace", gZenMarketplaceManager);
+  register_module("paneEpocaAccount", gEpocaAccount);
 
   // Restore the cached Firefox Labs nav button visibility so it shows
   // immediately when recipes are expected to be available, before
@@ -651,9 +664,9 @@ async function gotoPref(
   let redesignEnabled = srdSectionPrefs.all;
   let categories = document.getElementById("categories");
   const kDefaultCategoryInternalName = redesignEnabled
-    ? "paneSync"
+    ? "paneTabsBrowsing"
     : "paneGeneral";
-  const kDefaultCategory = redesignEnabled ? "sync" : "general";
+  const kDefaultCategory = redesignEnabled ? "tabsBrowsing" : "general";
   let hash = document.location.hash;
   let category = aCategory || hash.substring(1) || kDefaultCategoryInternalName;
 
